pkgs/phps.nix

```diff
@@ -43,8 +43,6 @@ let
 
               "--enable-libxml"
               "--with-libxml-dir=${prev.libxml2.dev}"
-            ]
-            ++ prev.lib.optionals (prev.lib.versionAtLeast args.version "7.3") [
               "--with-pcre-regex=${prev.pcre2.dev}"
```

Comment:
> checking for PCRE headers location... configure: error: Could not find pcre.h in /nix/store/r2cgszbs5mpjbrj0k28i9qa7wi7lqg15-pcre2-10.42-dev

Might be that older versions expect `${prev.pcre2.dev}/include`.

Reply:
Same issue

Reply:
We will need to look at the `configure.ac` or `m4` files, then.

Reply:
Then I'll happily leave this job to someone else, it's way beyond my knowledge at this moment.

Reply:
FYI, this issue is now:

![image](https://user-images.githubusercontent.com/252042/214870030-2071af21-d39a-4ad7-9c6d-3e0cb02bc875.png)

Reply:
Same issue:

```2023-01-26T15:34:06.3265322Z configure: error: Couldn't find pcre2-config```

I also noticed this, I don't know if it's related:

![image](https://user-images.githubusercontent.com/252042/214878561-4f1a198c-a915-400e-802b-56d226bc54a5.png)
2.

Reply:
As the `--with-external-pcre` is not recognized, the `--with-pcre-regex` is also added. So the error message is not relevant.

I'll take a look at the error next morning.

Reply:
I have succeeded this PR in #204 with fixes for this issue. Please take a look at it